### DISCUSSION TITLE
Output to a pulseaudio device an OBS source by adding filter

### DISCUSSION THREAD

**Discussion by ogmkp** [obsproject/obs-studio#5773] [General]
Hello.

I'm looking for a way to output the sound from an OBS source to a pulseaudio sound card as a filter.
Would you know a solution?

Thank you.

**Comment by Fenrirthviti** [comment]
We are not currently accepting support requests on GitHub Discussions. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for assistance.

Thank you!
